sos/report/plugins/__init__.py

```diff
@@ -1577,6 +1596,14 @@ def _add_cmd_output(self, **kwargs):
         pred = kwargs.pop('pred') if 'pred' in kwargs else None
         soscmd = SoSCommand(**kwargs)
         self._log_debug("packed command: " + soscmd.__str__())
+        for _skip_cmd in self.skip_cmds:
+            # This probably seems weird to be doing filename matching on the
+            # commands, however we want to remain consistent with our regex
+            # matching with file paths, which sysadmins are almost guaranteed
+            # to assume will use shell-style unix matching
+            if fnmatch.fnmatch(soscmd.cmd, _skip_cmd):
```

Comment:
Does it make sense to match against suggest_filename as well (if set)? I am not convinced on either yes or no here..

Reply:
I'd say no, to avoid potential clashes/confusion. But we can certainly keep an eye on this and add it later if need be.